nebula/ui/components/filterproxymodel.cpp

```diff
@@ -4,12 +4,9 @@
 
 #include "filterproxymodel.h"
 
-#include "logger.h"
-#include "qmlengineholder.h"
-
-namespace {
-Logger logger("FilterProxyModel");
-}
+#include <QDebug>
```

Comment:
Why the changes in this file?

Reply:
In order to move this class into the Nebula class, it was important to make sure that the C++ code doesn't depend on code in the `mozillavpn` target, otherwise this would result in a dependency loop. This mostly just required that we drop the use of the VPN's logging infrastructure and just do it in pure Qt.